app/controllers/api/v3x1/sources_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V3x1
+    class SourcesController          < Api::V3x0::SourcesController
+      def pause
+        src = Source.find(params.require(:source_id)).tap { |s| authorize(s) }
+        # the after_discard callback on the Application model handles discarding the source.
+        src.applications.each do |app|
+          app.discard!
+          AvailabilityMessageJob.perform_later("Application.pause", app, Insights::API::Common::Request.current_forwardable)
```

Comment:
in #369 you changed sending the app to the json format so it should be the same here

Reply:
Sure, updated and marked a dependency on that PR since it will have to go in first.